docs/source/components.rst

```diff
@@ -102,6 +111,14 @@ It is also possible to pass a dictionary of initial data to ``Form.__init__()``:
     form = BlogPostForm(initial=my_initial_data)
 
 
+When using a model form existing database values can also be loaded by passing a
+model instance as ``instance`` argument:
```

Comment:
I meant something like this:

```python
def my_view(request):
    form = MyForm(request.POST)
    if form.is_valid():
        MyModel.objects.create(foo=form.cleaned_data['foo'])
```
I intentionally didn't mention model forms because I wanted to emphasize that the validation is done by the model itself and not by the model form.